src/app/(proper_react)/redesign/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
```

Comment:
```suggestion
```

Could we maybe pass `l10n` in as a prop, so that this component is still a server component when included by a server component? To minimise the amount of code we have to run on the client side/